streaming/src/test/java/org/apache/spark/streaming/JavaAPISuite.java

```diff
@@ -1818,26 +1815,25 @@ public Integer call(String s) throws Exception {
   // InputStream functionality is deferred to the existing Scala tests.
   @Test
   public void testSocketTextStream() {
-      JavaReceiverInputDStream<String> test = ssc.socketTextStream("localhost", 12345);
+    ssc.socketTextStream("localhost", 12345);
   }
 
-  @Test
-  public void testSocketString() {
-
-    class Converter implements Function<InputStream, Iterable<String>> {
```

Comment:
Any specific reason to pull this class out of the method? It doesn't seem to be used anywhere else.